api/v1beta1/metallb_webhook.go

```diff
@@ -110,5 +111,11 @@ func (metallb *MetalLB) validate() error {
 		metallb.Spec.BGPBackend != params.FRRMode {
 		return errors.New("Invalid BGP Backend, must be one of native, frr, frr-k8s")
 	}
+
+	if metallb.Spec.BGPBackend != params.FRRK8sMode &&
+		metallb.Spec.FRRK8SConfig != nil {
+		return fmt.Errorf("can't apply frrk8s config while running in %s mode", metallb.Spec.BGPBackend)
+	}
+
```

Comment:
let's add a validation that the cidrs are valid to be safe?

Reply:
done